src/components/v5/shared/StatusText/StatusText.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FC } from 'react';
+import { useIntl } from 'react-intl';
+
+import clsx from 'clsx';
+import { StatusTextProps } from './types';
+import Icon from '~shared/Icon';
+import { STATUS_TYPES } from './consts';
+
+const displayName = 'v5.shared.StatusText';
+
+const StatusText: FC<StatusTextProps> = ({
+  title,
+  status,
+  withIcon = true,
+  textClassName = 'text-md',
+}) => {
+  const { formatMessage } = useIntl();
+
+  const titleText = typeof title === 'string' ? title : formatMessage(title);
```

Comment:
Can we avoid passing `MessageDescriptior` to components? Use `React.ReacNode` or just `string`

Reply:
It's children now